static/js/fx-bento.js

```diff
@@ -20,10 +20,44 @@ function createAndAppendEl(wrapper, tagName, className = null) {
 
 async function getlocalizedBentoStrings() {
 
-  const localizedBentoStrings = {
-    "bentoButtonTitle":"Firefox apps and services","bentoHeadline":"Firefox is tech that fights for your online privacy.","bentoBottomLink":"Made by Mozilla","fxDesktop":"Firefox Browser for Desktop","fxLockwise":"Firefox Lockwise","fxMobile":"Firefox Browser for Mobile","fxMonitor":"Firefox Monitor","pocket":"Pocket","fxSend":"Firefox Send","mobileCloseBentoButtonTitle":"Close menu",
-  };
+  const renderedStrings = document.querySelector(".bento-strings");
+
+  if (!renderedStrings) {
+    return {
+      "bentoButtonTitle":"Firefox apps and services",
+      "bentoHeadline":"Firefox is tech that fights for your online privacy.",
+      "bentoBottomLink":"Made by Mozilla",
+      "fxDesktop":"Firefox Browser for Desktop",
+      "fxLockwise":"Firefox Lockwise",
+      "fxMobile":"Firefox Browser for Mobile",
+      "fxMonitor":"Firefox Monitor",
+      "pocket":"Pocket",
+      "fxSend":"Firefox Send",
+      "mobileCloseBentoButtonTitle":"Close menu",
```

Comment:
Should this be `bentoButtonCloseLabel` like the other references?

Reply:
Yup. Pocket too. Good catch. This was the old block of copy. I'll update the string refs to match (and test it failing)

Reply:
Updated all of them. Good to go!